### ISSUE TITLE
Indexing fails with ollama:nomic-embed-text

### ISSUE THREAD

**Issue by hoblin** [continuedev/continue#1435]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: macOS 14.5
- Continue: 0.8.24 - 2024-04-12
- IDE: VSCode 1.90.0
```

### Description

Indexing freezes at 25% for a while (popup displays "Completed indexing chunks) then fails with the error message:
```
Error updating the vectordb::nomic-embed-text index: Error: Invalid argument error: Values length 0 is less than the length (768) multiplied by the value size (768) for FixedSizeList(Field { name: "item", data_type: Float32, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }, 768)
```

Config:
```json
{
  "models": [
    {
      "title": "Codestral",
      "provider": "ollama",
      "model": "codestral"
    },
    {
      "title": "Ollama",
      "provider": "ollama",
      "model": "AUTODETECT"
    }
  ],
  "customCommands": [
    {
      "name": "test",
      "prompt": "{{{ input }}}\n\nWrite a comprehensive set of unit tests for the selected code. It should setup, run tests that check for correctness including important edge cases, and teardown. Ensure that the tests are complete and sophisticated. Give the tests just as chat output, don't edit any file.",
      "description": "Write unit tests for highlighted code"
    }
  ],
  "tabAutocompleteModel": {
    "title": "Codestral",
    "provider": "ollama",
    "model": "codestral"
  },
  "allowAnonymousTelemetry": true,
  "embeddingsProvider": {
    "provider": "ollama",
    "model": "nomic-embed-text"
  }
}
```

### To reproduce

1. Install Codestral
2. Pull the ollama models
3. Shut down the VSCode
4. Remove files except the config from ~/.continue directory for a fresh start
5. Open the VSCode with `code .` from the project's directory

### Log output

```Shell
Error updating the vectordb::nomic-embed-text index: Error: Invalid argument error: Values length 0 is less than the length (768) multiplied by the value size (768) for FixedSizeList(Field { name: "item", data_type: Float32, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }, 768)
```

**Comment by hoblin** [comment]
I've got a similar error with `mxbai-embed-large`:
```
Error updating the vectordb::mxbai-embed-large index: Error: Invalid argument error: Values length 0 is less than the length (1024) multiplied by the value size (1024) for FixedSizeList(Field { name: "item", data_type: Float32, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }, 1024)
```

**Comment by sestinj** [comment]
@hoblin thanks for sharing this. I've seen it from others too, it looks like Ollama embeddings in general are failing. I'll have an update soon

**Comment by Cheizr** [comment]
Experiecing this also today, was working fine 1 hour ago.

![image](https://github.com/continuedev/continue/assets/22793954/e1f5fe3c-4331-4f27-80db-d020fac1fabd)

**Comment by rohhro** [comment]
It's not just Ollama with nomic-embed-text fails,  OpenAI embeddings also fail.

**Comment by boshk0** [comment]
I can confirm it fails with nomic-embed-text too.

**Comment by sestinj** [comment]
The PR looks to have solved it, but I'll wait for confirmation. This is going to be released in the next pre-release version (0.9.160)
